### DISCUSSION TITLE
Any hints for #166?

### DISCUSSION THREAD

**Discussion by Mrdigs** [cssbattle/forum#368] [General]
I'm a new member who's been enjoying the battles immensely, and I'm happy with my performance so far, clocking in at place 50-80 on the majority of targets I've attempted.

But #166 has me totally bamboozled. I haven't attempted it yet, but looking at it, I can't see how it could ever be possible in less 600-700 characters.

The best plan I've come up with is 5 columns of "fingers" with rounded tips, maybe using box-reflect below on each to keep it down to 5 elements rather than 10, overlaid with a semi circle at the bottom using an extra element.

I can see that some of it can be done with multiple background gradients, but these are going to add up quickly in characters.

There has to be something I'm missing. The battle is titled "initial" which gives a clue but I haven't found a use for that in any of the other targets in the battle.

Is anybody tempted to give me a clue, however cryptic, to put me on the right path?

Thanks in advance!

**Comment by PavButsa** [comment]
Hello!
For this solution, I used only `<BODY>` element with 9 shadows.
To explain my idea, I temporarily shrunk `<BODY>` and moved it down. I also changed colors of some shadows and made an outline around `<BODY>` — just to make things clear.

![css_battle_166_hint_1](https://github.com/cssbattle/forum/assets/8935771/3aad1f99-7f7e-4961-849f-6a6990e64378)

Then we add `clip-path: inset(-193% -283% round 9in)` and get this:

![css_battle_166_hint_2](https://github.com/cssbattle/forum/assets/8935771/b0071466-61c7-4dcd-b742-0977e61e7989)

Now we should enlarge `<BODY>` and move it up.
With this idea, I got 232 characters.

By the way, titles of battles don't have a special meaning, so the title "Initial" doesn't mean that the CSS value "initial" plays a role in this battle.
